### PR TITLE
pfed-sso: redirect to login on failure sso

### PR DESCRIPTION
**Pull request by amithkb**
- Redirect to default login whenever pingfed fails to provide auth token or fails due to some other IO issue

### CODE REVIEW

File: server/plugins/pfed-sso/src/main/java/com/walmartlabs/concord/server/plugins/pfedsso/RedirectHelper.java [hunk 1]
```diff
@@ -42,6 +45,11 @@ public void sendRedirect(HttpServletResponse response, String location) throws I
         response.sendRedirect(response.encodeRedirectURL(normalizeLocation(location)));
     }
 
+    public void redirectToLoginOnError(HttpServletResponse response, String errorMsg) throws IOException {
+        log.warn("Error during sso login -> " + errorMsg);
```
Comment (ibodrov) [inline server/plugins/pfed-sso/src/main/java/com/walmartlabs/concord/server/plugins/pfedsso/RedirectHelper.java:49]: Use placeholders `{}` please.

File: server/plugins/pfed-sso/src/main/java/com/walmartlabs/concord/server/plugins/pfedsso/SsoLogoutFilter.java [hunk 2]
```diff
@@ -52,7 +56,11 @@ public void doFilter(HttpServletRequest request, HttpServletResponse response, F
         String initParam = request.getParameter("init");
         boolean isInit = initParam != null && !initParam.trim().isEmpty();
         if (isInit) {
-            ssoClient.revokeToken(SsoCookies.getRefreshCookie(request));
+            try {
+                ssoClient.revokeToken(SsoCookies.getRefreshCookie(request));
+            } catch (Exception e) {
+                log.warn("Error in revoking sso token during logout -> {}", e.getMessage(), e);
```
Comment (ibodrov) [inline server/plugins/pfed-sso/src/main/java/com/walmartlabs/concord/server/plugins/pfedsso/SsoLogoutFilter.java:62]: Use placeholders `{}` please.

**Review by ibodrov — APPROVED** [review]

File: server/plugins/pfed-sso/src/main/java/com/walmartlabs/concord/server/plugins/pfedsso/SsoLogoutFilter.java (hunk 2 above)
Comment (amithkb) [reply server/plugins/pfed-sso/src/main/java/com/walmartlabs/concord/server/plugins/pfedsso/SsoLogoutFilter.java]: updated

File: server/plugins/pfed-sso/src/main/java/com/walmartlabs/concord/server/plugins/pfedsso/RedirectHelper.java (hunk 1 above)
Comment (amithkb) [reply server/plugins/pfed-sso/src/main/java/com/walmartlabs/concord/server/plugins/pfedsso/RedirectHelper.java]: updated

**Review by ibodrov — APPROVED** [review]

**Pull request merged**